src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar-wrapper {
+  display: flex;
+  padding: 100px 100px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  width: calendarWidth($calendarWidth, $calendarPadding);
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendarGap;
+
+  @each $day, $startDay in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: (($calendarWidth + 1) * $startDay);
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    @include daySize($calendarWidth);
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+    transition: all 0.5s;
```

Comment:
It's not recommended to set 'all' as transition property. Better specify the exact properties 
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/70be6439-add6-49bb-978e-511864e97a95)